.github/CONTRIBUTING.md

```diff
@@ -35,14 +41,14 @@ See [CLI Local Development](../packages/cli#local-development) for more details.
 
 ## Verifying your change
 
-Once you are done with your changes (we even suggest doing it along the way), make sure all the tests still pass by running:
+Once you are done with your changes (we even suggest doing it along the way), make sure the relevant tests still pass by running the relevant tests. Example:
 
 ```
-pnpm test-unit
+# from root of this project
+cd packages/cli
+pnpm test test/integration-1.test.ts
```

Comment:
Shouldn't turbo run the relevant tests automatically?

I think this should mention you are only running a specific test file.

And you can run from the root to run all relevant tests.